apps/glusterfs/volume_entry.go

```diff
@@ -418,11 +418,34 @@ func (v *VolumeEntry) cleanupCreateVolume(db wdb.DB,
 
 	// from a quick read its "safe" to unconditionally try to delete
 	// bricks. TODO: find out if that is true with functional tests
-	DestroyBricks(db, executor, brick_entries)
+	reclaimed, err := DestroyBricks(db, executor, brick_entries)
+	if err != nil {
+		logger.LogError("failed to destory bricks: %v", err)
```

Comment:
I'd like to tweak the message to say something like "failed to destroy bricks during cleanup" or similar to really highlight that we already were in a failure path.